store/policy.go

```diff
@@ -78,6 +78,78 @@ func (s *Store) GetPolicy(ctx context.Context, find *api.PolicyFind) (*api.Polic
 	return policy, nil
 }
 
+// DeletePolicy deletes an existing ARCHIVED policy by PolicyDelete.
+func (s *Store) DeletePolicy(ctx context.Context, delete *api.PolicyDelete) error {
+	// Validate policy.
+	// Currently we only support PolicyTypeSchemaReview type policy to delete by id
+	if delete.Type != api.PolicyTypeSchemaReview {
+		return &common.Error{Code: common.Invalid, Err: fmt.Errorf("Invalid policy type")}
+	}
+
+	tx, err := s.db.BeginTx(ctx, nil)
+	if err != nil {
+		return FormatError(err)
+	}
+	defer tx.PTx.Rollback()
+
+	find := &api.PolicyFind{
+		ID:   &delete.ID,
+		Type: &delete.Type,
+	}
+	policyRawList, err := findPolicyImpl(ctx, tx.PTx, find)
```

Comment:
Do we need to do a Find? Maybe we can let deletePolicyImpl() to handle it.

Reply:
Please check the comment here https://github.com/bytebase/bytebase/pull/1144#discussion_r856085374